echo-model/src/main/java/com/netflix/spinnaker/echo/model/Trigger.java

```diff
@@ -128,6 +128,7 @@ public String toString() {
   boolean dryRun = false;
 
   List<Map<String, Object>> notifications;
+  List<Map<String, String>> artifacts;
```

Comment:
I think you want `<String, Object>` because of the `metadata` field.

Reply:
Good point, I'll fix that